packages/wonder-blocks-button/components/button-core.js

```diff
@@ -84,14 +83,22 @@ export default class ButtonCore extends React.Component<Props> {
         ];
 
         const commonProps = {
-            "aria-disabled": disabled ? "true" : undefined,
-            "aria-label": ariaLabel,
             "data-test-id": testId,
             role: "button",
             style: [defaultStyle, style],
             ...handlers,
         };
 
+        if (commonProps["aria-disabled"]) {
+            // eslint-disable-next-line no-console
+            console.warn(
```

Comment:
I wonder if we should make this a lot angrier? Since this is a problem if it gets to the client. I'd be open to making this: `throw new Error("wb-button: ...});` and removing the logic after it. We want to make sure people don't do this!

Reply:
I think we should either complain (in which case we throw an exception) or we make `aria-disabled` do the right thing (deleting the original value and setting the `disabled` prop to be the right value).

Reply:
I think I'd err on the side of making it do the right thing and complaining if the two values don't match up for some reason.

Reply:
I think we might be able to use flow's `$Rest` utility type to construct `Prop`s that omit `aria-disabled` for components have a `disabled` prop, e.g.
```
type Props = {|
   ...$Rest<AriaProps, {| "aria-disabled": boolean |},
   disabled: boolean,
   ...
|}
```

Reply:
@kevinbarabash Tried with `$Rest` and it works only if the prop is required, but the problem is that all `AriaProps` attributes are optional. I have to use `"aria-disabled"?` to map this key to the original `AriaProps` object. Not sure if I'm missing something on the implementation. 

`...$Rest<AriaProps, {| "aria-disabled"?: "false"|"true" |}>,`

Reply:
That makes sense.  I forgot about the`"false"|"true"` that used in `AriaProps`.

Reply:
There's a note in the [docs](https://flow.org/en/docs/types/utilities/#toc-diff) for `$Diff` (which is the inverse? of `$Rest`) on how to get around the issue of optionality:
```
type A = $Diff<{}, {nope: number}>; // Error
type B = $Diff<{}, {nope: number \| void}>; // OK
```
I tried the following and flow accepts it and finds the error:
```
type Props = {|
    ...$Rest<AriaProps, {| "aria-disabled": "true" | "false" | void |}>,
    disabled: boolean,
|};

const props: Props = {
    "aria-disabled": "true",
    disabled: true,
};
```

Reply:
I wish Flow had TypeScript's `Omit` helper which lets you do:
```
type Props = Omit<AriaProps, "aria-disable"> & { disabled: boolean, ... };
```
https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/#the-omit-helper-type

One thing `$Rest` has going for it is that you can remove multiple properties.